bleak/backends/bluezdbus/client.py

```diff
@@ -125,7 +125,8 @@ async def connect(self, dangerous_use_bleak_cache: bool = False, **kwargs) -> bo
         # Each BLE connection session needs a new D-Bus connection to avoid a
         # BlueZ quirk where notifications are automatically enabled on reconnect.
         self._bus = await MessageBus(
-            bus_type=BusType.SYSTEM, negotiate_unix_fd=True
+            bus_type=BusType.SYSTEM,
+            negotiate_unix_fd=not BlueZFeatures.has_mtu_property,
```

Comment:
`negotiate_unix_fd` is also needed when `BlueZFeatures.write_without_response_workaround_needed`

Reply:
I've also been mulling over possibly using "AcquireNotify" to improve performance of notifications which would require passing file descriptors.

Reply:
has_mtu_property is always higher but probably good to have an explicit check 👍

Reply:
25219d00fab760dba0a8257ea6895288352e90a7